src/Compilers/CSharp/Portable/Symbols/Synthesized/GeneratedNames.cs

```diff
@@ -439,6 +439,43 @@ internal static string AsyncBuilderFieldName()
             return "<>t__builder";
         }
 
+        internal static string DelegateCacheContainerType(int generation, string? methodName = null, int methodOrdinal = -1, int ownerUniqueId = -1)
+        {
+            const char NameKind = (char)GeneratedNameKind.DelegateCacheContainerType;
+
+            var result = PooledStringBuilder.GetInstance();
+            var builder = result.Builder;
+
+            builder.Append('<').Append(methodName).Append('>').Append(NameKind);
+
+            if (methodOrdinal > -1)
+            {
+                builder.Append(GeneratedNameConstants.SuffixSeparator).Append(methodOrdinal);
+            }
+
+            if (ownerUniqueId > -1)
+            {
+                builder.Append(IdSeparator).Append(ownerUniqueId);
+            }
+
+            if (generation > 0)
+            {
```

Comment:
It seems that other generated names have an identifying letter after the `>` based on `GeneratedNameKind`. We do that even for members of types with generated names (such as fields of anonymous types). 
So I think we should add a char here too.

Reply:
I did notice that. And it seems we're running out of ASCII chars. The container only holds cache fields, it looks unnecessary to allocate another char for fields? I don't know. Is this char served as other purposes?

Reply:
I don't know if running out of single ascii characters is a big concern (we can use double ascii characters as needed).
That said, I'd be okay to leave as-is, maybe with a comment.
@AlekseyTs Any thoughts either way?

Reply:
>  thoughts either way?

I am not sure why we should care about anything, but uniqueness here. As far as I understand, we are not trying to parse the names back and we are not trying to reuse any of the symbols across generations.

Reply:
Alright, let's leave as-is then.